src/server/multi_command_squasher.cc

```diff
@@ -22,6 +22,14 @@ template <typename F> void IterateKeys(CmdArgList args, KeyIndex keys, F&& f) {
     f(args[*keys.bonus]);
 }
 
+bool CheckConnStateClean(const ConnectionState& state) {
+  DCHECK_EQ(state.exec_info.state, ConnectionState::ExecInfo::EXEC_INACTIVE);
+  DCHECK(state.exec_info.body.empty());
+  DCHECK(!state.script_info);
+  DCHECK(!state.subscribe_info);
+  return true;
```

Comment:
I find it odd to return `true` unconditionally. Am I missing something?

Reply:
🤷🏻‍♂️ I just wanted to group the DCHECKs in a separate function. I want to call it only for debug builds, so I wrap it inside a DCHECK -> expects a bool reply. I can call it unconditionally